src/common/wasm_util.h

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#pragma once
+
+#include <string_view>
+#include <vector>
+#include <unordered_map>
+
+namespace proxy_wasm {
+namespace common {
+
+// Utilitiy functions which directly operate on Wasm bytecodes.
+class WasmUtil {
```

Comment:
Maybe rename it to `BytecodeUtil` or `BytecodeParser`? `Wasm` alone isn't descriptive enough in this project.

Reply:
renamed to BytecodeUtil. Thanks!